### PR TITLE
attempt to fix PR#40

### PR DESCRIPTION
**Pull request by DominikNowak**
closes #40

### CODE REVIEW

**Review by Rednaxela700 — CHANGES_REQUESTED** [review]
This code is a mess, I hope it will be cleaned up quickly 🧽 

Steps to take:
1. Reorganize
2. Cleanup
3. Double check 
4. Push

Also please keep commit messages explanatory. I'd like to know the process, see more than a single commit

File: src/app/components/ModeMenu.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  });
+  const menuOption = {
+    selectOption: '',
+  };
+  const menuElements = document.querySelectorAll('.option_button');
+  function handleSelectOption() {
+    menuOption.selectOption = this.dataset.option;
+    menuElements.forEach((optionBtn) => {
+      const btnOption = optionBtn;
+      btnOption.style.color = 'rgba(0, 0, 0, 0.4)';
```
Comment (vieraboschkova) [inline src/app/components/ModeMenu.js:26]: question concerning the app itself - can we define if we use rgb or hex for color definitions?

File: styles/components/_modeMenu.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      text-align: center;
+      font-size: 36px;
+      color: rgba(0, 0, 0, 0.4);
+      font-weight: 600;
+      background-color: #ffffff;
+      border: none;
+      outline: none;
+      cursor: pointer;
+      padding-bottom: 20px;
+      font-family: $font-primary;
```
[the remark this answers is not in the document]
Comment (vieraboschkova) [reply styles/components/_modeMenu.scss:28]: Thanks, that is something i never before organized in my code, although I tried...  So by type should be ok?
/* Positioning */
/* Display & Box Model */
/* Color */
/* Text */
 /* Other */

**Review by paula0403 — APPROVED** [review]

File: styles/components/_modeMenu.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    color: rgba(0, 0, 0, 0.4);
+    font-family: $font-primary;
+    text-align: center;
+    font-size: 36px;
+    color: rgba(0, 0, 0, 0.4);
+    font-weight: 600;
+    outline: none;
+    cursor: pointer;
+  }
+  &_button--select {
```
Comment (vieraboschkova) [inline styles/components/_modeMenu.scss:25]: Maybe the structure could look like that: 
.menu {
  &__button {
    &--select{
      &::after{
      }
    }

**Review by vieraboschkova — COMMENTED** [review]
check the scss structure

**Review by vieraboschkova — APPROVED** [review]

**Review by Rednaxela700 — APPROVED** [review]

**Pull request merged**